controllers/book.js

```diff
@@ -56,6 +58,20 @@ const updateBook = async (req, res) => {
     console.error(err);
     res.status(500).send({ message: 'Server error' });
   }
-}
+};
+
+const deleteBook = async (req, res) => {
+  const admin = await checkAdmin(req, res);
+  if (!admin) return res.status(403).send({ message: 'Forbidden' });
 
-module.exports = { getBooks, createBook, updateBook };
+  try {
+    const book = await Book.findByIdAndDelete({ _id: req.params.id });
+    if (!book) {
+      res.status(404).send({ message: `No job with id ${req.params.id}` });
```

Comment:
mama, you don go use mistake add your job api project code. No job with id..lmao @khareemah